### PR TITLE
Update AT templates for all products

### PR DESCRIPTION
**Pull request by shawndwells**
- Everything is still not applicable
- Updated answers to reflect control breakouts, eg AT-1(a), AT-1(b)

### CODE REVIEW

File: coreos-4/policies/SA-System_and_Services_Acquisition/component.yaml [hunk 1]
```diff
@@ -2,42 +2,123 @@
...
   narrative:
-    - text: |
-        'This control reflects organizational procedure/policy and is not
-        applicable to component-level configuration.'
+    - key: a
+      text: |
+        Organizational processes to determine information security requirements
+        for the information system or information system service in
+        mission/business process planning are outside the scope of
+        CoreOS configuration.
```
[the remark this answers is not in the document]
Comment (shawndwells) [reply coreos-4/policies/SA-System_and_Services_Acquisition/component.yaml]: Fixed

File: ansible-tower/policies/AT-Awareness_and_Training/component.yaml [hunk 2]
```diff
@@ -8,96 +8,124 @@
   covered_by: []
   implementation_status: not applicable
   narrative:
-    - text: |
-        'This control reflects organizational procedure/policy and is not
-        applicable to component-level configuration.'
+    - key: a
+      text: |
+        This control reflects organizational procedure/policy and is not
+        applicable to component-level configuration.
```
Comment (redhatrises) [inline ansible-tower/policies/AT-Awareness_and_Training/component.yaml]: @shawndwells should we say `${product_name}` instead of `component-level`?

Comment (shawndwells) [reply ansible-tower/policies/AT-Awareness_and_Training/component.yaml]: Yes, probably.

The current language is from a template. 

What about using the template for now, and we can ``s/component-level/${product}/g`` once there are control responses?

Comment (redhatrises) [reply ansible-tower/policies/AT-Awareness_and_Training/component.yaml]: @shawndwells sure. why not.

**Pull request merged**